Makefile

```
@@ -0,0 +1,15 @@
+CC ?= gcc
```

```suggestion
CC ?= cc
```

Removes the dependencies for Mac

---

> ```suggestion
> CC ?= cc
> ```
> 
> Removes the dependency of gcc for Mac

Mac has `gcc`, its an alias to `clang`.

---

so? On my machine, CC is set to `cc` already. Then we compile with `cc` on this machine.

---

that's the magic of `?=`